CONTRIBUTING.md

```diff
@@ -38,7 +38,14 @@ echo "Test the CLI."
 yarn run smoke
 echo "Test a fresh local install (see below)"
 echo "Test the extension"
+
 echo "Test the lighthouse-viewer build"
+# Manual test for now:
```

Comment:
I had it there but I think it makes more sense to keep next to the reminder echo.